homeassistant/components/hdmi_cec/__init__.py

```diff
@@ -374,10 +374,27 @@ class CecEntity(Entity):
     def __init__(self, device, logical) -> None:
         """Initialize the device."""
         self._device = device
-        self._icon = None
         self._state: str | None = None
         self._logical_address = logical
         self.entity_id = "%s.%d" % (DOMAIN, self._logical_address)
+        self._attr_should_poll = False
```

Comment:
This can be a class variable

Reply:
? I do not understand what you mean: self._attr_should_poll is a class variable defined in entity.py line 253.

Reply:
Yes, but you can define this as class variable as well

```py
class MyEntity(SensorEntity):
   """Comment..."""

   _attr_should_poll = False

   def __init__(self):....
```

Reply:
Thanks for the idea, I thought it overwriting the inherited definition could be a problem (with typing etc). It is corrected !

and I will remember it for other PRs.